service/api_task_test.go

```diff
@@ -215,6 +215,7 @@ func TestAssignNextAvailableTaskWithPlannerSettingVersionLegacy(t *testing.T) {
 				So(err, ShouldBeNil)
 				So(t, ShouldBeNil)
 			})
+			// STU: Hmmm - in what sense does this demonstrate that we "continue"?
```

Comment:
I think this means that we don't return an error. But yeah, it doesn't demonstrate that the next task after the non-existent task would be dispatched.